students/alex_whitty/lesson3/list_lab_series2.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python
+
+fruits = ['apples', 'pears', 'oranges', 'peaches']
+print(fruits)
+
+fruits.remove('peaches')
+print(fruits)
+
+remove_fruit = input("Which fruit would you like to remove?>>>")
+if remove_fruit == "apples":
+    fruits.remove('apples')
+    print(fruits)
+elif remove_fruit == "pears":
+    fruits.remove('pears')
+    print(fruits)
+elif remove_fruit == "oranges":
+    fruits.remove('oranges')
+    print(fruits)
+else:
+    print("Item not found!")
```

Comment:
```
remove_fruit = input("Which fruit would you like to remove?>>>")
for fruit in fruits[:]:  # we iterate over a copy of the list because we are changing the original
    if remove_fruit.lower() == fruit.lower():
         fruits.remove(remove_fruit.lower())
```